core/provider_management.py

```
@@ -0,0 +1,653 @@
+                amount=pricing_data.get('amount', Decimal('0')),
+                available=pricing_data.get('available', True),
+                effective_date=pricing_data.get('effective_date', timezone.now().date())
+            )
+            
+            return {
+                'success': True,
+                'message': 'Service added to provider successfully',
+                'hospital_service_id': str(hospital_service.id)
+            }
+            
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)